src/main/java/com/uci/adapter/firebase/web/FirebaseNotificationAdapter.java

```diff
@@ -254,6 +256,8 @@ public Mono<List<XMessage>> apply(List<XMessage> msgList) {
                                 } else {
                                     log.error("FirebaseNotificationAdapter:processOutBoundMessageF::AdapterId not found : " + nextMsg);
                                 }
+                                nextMsg.setTo(to);
+                                nextMsg.setMessageId(MessageId.builder().channelMessageId(channelMessageId).build());
```

Comment:
What are these changes in regard to?

Reply:
These changes are for notification message id which we created manually and sent to the Android or fcm side and this message id android is using for delivery report.